pkg/healthcheck/healthcheck.go

```diff
@@ -281,3 +281,41 @@ func (lb *LbStatusUpdater) UpsertServer(u *url.URL, options ...roundrobin.Server
 	}
 	return err
 }
+
+// BalancerHandlers is a list of BalancerHandler(s) that implements BalancerHandler.
+type BalancerHandlers []BalancerHandler
+
+func (b BalancerHandlers) ServeHTTP(w http.ResponseWriter, req *http.Request) {
```

Comment:
This is weird, but I don't know yet if it is a problem. I need to think about it.

Reply:
alternative implementation that removes the need for that empty ServeHTTP: 
https://github.com/mpl/traefik/tree/healthcheck-lb
(clean commit on top of the last one in this PR)

Reply:
I pushed that commit to the PR, as agreed by chat with @jbdoumenjou